Sources/HipMobileUI/Pages/ExhibitRouteDownloadPage.xaml

```diff
@@ -56,35 +55,36 @@
                             </ProgressBar.Effects>
                         </ProgressBar>
                         <Label Text="{Binding LoadingProgress, StringFormat='{0:p0}'}"
-                               HorizontalTextAlignment="Center" FontSize="16" />
+                               HorizontalTextAlignment="Center" FontSize="16" Style="{DynamicResource Ssp-Italic}"/>
                     </StackLayout>
 
                     <StackLayout x:Name="ButtonView" Grid.Row="3" Orientation="Vertical" VerticalOptions="End" Padding="0">
                         <BoxView HeightRequest="1" Color="{StaticResource DarkGrayColor}" />
                         <StackLayout Orientation="Horizontal" Padding="8" BackgroundColor="{StaticResource LightGrayColor}">
-                            <Label IsVisible="{Binding DownloadFinished}" Text="{helpers:Translate ExhibitRouteDownloadPage_Overview}" FontSize="20"
-                                   TextColor="{StaticResource AccentColor}"
+                            <Label IsVisible="{Binding DownloadFinished}" Text="Übersicht" FontSize="20"
```

Comment:
`Text`-attributes should always be set with strings from the Strings.resx file. Daniel worked on this so this is probably only trying to override his changes. Just revert it.